megamek/src/megamek/client/ui/swing/tileset/EntityImage.java

```diff
@@ -197,7 +197,13 @@ public EntityImage(Image base, Image wreck, Camouflage camouflage, Component com
         decal = getDamageDecal(entity, secondaryPos);
         smoke = getSmokeImage(entity, secondaryPos);
         unitHeight = entity.height();
-        unitElevation = entity.getElevation();
+        int elevation = 0;
+        try {
+            elevation = entity.getElevation();

```

Comment:
Do we really want to use exceptions for flow control here?

Reply:
Doesn't look like the place for an exception. This was old code in Entity. I removed the exception. One could add a hasElevation or knowsElevation method or something but that's a bit beyond this PR.